packages/docs/src/components/content/MaturityChecklist/MaturityChecklist.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
 import MaturityChecklistItem, { CheckStatus } from './MaturityChecklistItem';
+import { Link } from 'gatsby';
```

Comment:
I believe this can be removed, not being used.

Reply:
Good catch!